ads/_config.js

```diff
@@ -838,6 +838,8 @@ export const adConfig = {
     renderStartImplemented: true,
   },
 
+  'rnetplus': {},
```

Comment:
FYI, preload/prefetch are popular options here if you know your creative will be requesting a certain file

Reply:
Thank you, but from the technical manner we have JSONP request and it cannot be precached. The images and other stuff are relied to server response and we cannot prefetch it too... Anyway I'll review code to think if we still have something to declare, thank you.

Reply:
Sounds good.